### ISSUE TITLE
Missing relationships when using `Base`

### ISSUE THREAD

**Issue by maxcountryman** [eralchemy/eralchemy#67]
Hi,

I'm noticing an issue where when I provide a `Base` the rendered PNG does not map the `ForeignKey` relationships.

As an example, I have a Facebook table declared with a `campaign_id` column like so:

```python
...
    campaign_id = sa.Column(
        sa.BigInteger(),
        sa.ForeignKey(
            "facebook.campaigns.id",
            name="campaign_hourly_stats_campaign_id_fkey",
        ),
    )
...
```

However the rendered ERD doesn't recognize this relationship:

<img width="325" alt="Screen Shot 2019-10-01 at 9 29 36 AM" src="https://user-images.githubusercontent.com/74351/65981470-0846d880-e42e-11e9-9e87-cc9b882518f8.png">

Any idea on why this might be?

**Comment by maxcountryman** [comment]
So the underlying issue is [this line](https://github.com/Alexis-benoist/eralchemy/blob/d6fcdc67d6d413bb174bf008fd360044e1dff5a7/eralchemy/sqla.py#L18)--basically the schema name is missing for the `left_col` property which causes all the relationships to be stripped during processing. It's easy enough to add this, but I'm not sure how that might break other schema.

**Comment by henhuy** [comment]
I could make it work using #70 - thanks @psipika!
Any progress in merging this PR?

**Comment by maurerle** [comment]
This should now be possible using `$ eralchemy -i 'postgresql+psycopg2://username:password@hostname:5432/databasename' -s "public, facebook"`

fixed in 9fbd13a767e828f372e23dbf5a15091df8bf0d42
